pkg/constants/constants.go

```diff
@@ -4,11 +4,13 @@ const (
 	HTML     = "html"
 	Text     = "text"
 	Markdown = "markdown"
+	Post     = "post"
```

Comment:
I think we can add a comment here to explain what is post format, maybe with the link below
https://open.feishu.cn/document/uAjLw4CM/ukTMukTMukTM/im-v1/message/create_json#45e0953e